code-samples/serving/websockets-go/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Use the official Golang image to create a build artifact.
+# This is based on Debian and sets the GOPATH to /go.
+FROM golang:latest as builder
+
+ARG TARGETOS
```

Comment:
Tested locally works as epxected:
```
$ wscat --connect 127.0.0.1:8080/ws
Connected (press CTRL+C to quit)
> hello
< hello
> hi
< hi
> 
```

Reply:
To be consistent with the other samples eg. grpc we need to add something like
```
docker run --rm {username}/grpc-ping-go \
  /client \
  -server_addr="grpc-ping.default.1.2.3.4.sslip.io:80" \
  -insecure
```

Reply:
never done that before. I mostly do never use `docker`, I just run things w/ `ko` 

feel free to add things to this post.

Reply:
@skonto 

```
docker run --rm docker.io/matzew/ws-goss \      
  /server \                                                 
  -server_addr="websockets-go.default.1.2.3.4.sslip.io:80" \
  -insecure
```

I am not getting this to work, please feel free to add the correct content. Otherwise we merge this and deal with this later?

The section above shows how to the binary w/in kube already. Not really adding too much value

Reply:
I could do `go run cmd/server/main.go` ... 

and localhost - that is simple enought - and has similar value

Reply:
It is ok let's fix this in #6110